src/esp/assets/ResourceManager.cpp

```diff
@@ -750,29 +765,24 @@ bool ResourceManager::loadPTexMeshData(const AssetInfo& info,
         node.addFeature<gfx::PTexMeshDrawable>(*pTexMeshData, jSubmesh,
                                                shaderManager_, drawables);
 
-        if (computeAbsoluteAABBs_) {
-          staticDrawableInfo.emplace_back(
-              StaticDrawableInfo{node, static_cast<uint32_t>(jSubmesh)});
-        }
+        staticDrawableInfo.emplace_back(
+            StaticDrawableInfo{node, static_cast<uint32_t>(jSubmesh)});
       }
     }
-    if (computeAbsoluteAABBs_) {
```

Comment:
Let's get rid of computeAbsoluteAABBs_ from loadPTexMeshData arg list here, since we don't use it anyway for PTEX.